notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb

```diff
@@ -98,7 +98,7 @@
       "source": [
```

Comment:
<p>dataset_uri not used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
<p>I've removed the parameter. The Dataset artifact doesn't seem to be used at all, and the lineage it generates isn't useful in this pipeline.</p>